### DISCUSSION TITLE
Although it was submitted and approved on October 18th (JST), as of October 22nd (JST), they still have not granted me access to the GitHub Student benefits.

### DISCUSSION THREAD

**Discussion by code-onigiri** [community/community#142195] [GitHub Education] [answered]
### Hi everyone,

As the title says,
I hope to gain something here.

**Comment by davevad93** [comment]
Hi @code-onigiri ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @code-onigiri was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by Dipeshdahiya** [comment, marked as answer]
Hi @code-onigiri,

It seems like this kind of delay is not uncommon based on the discussions you've referenced. As mentioned, the status label should change to purple when the benefits become available, but the waiting time can vary depending on factors like request volume. It’s good to know that while it usually takes just a few days, sometimes it can take longer, up to a couple of weeks.

For anyone in a similar situation, checking out the other discussions and the Student Developer Pack Application FAQs might provide more insight into the waiting times and what to expect.

Hopefully, your benefits will be granted soon!
